### ISSUE TITLE
Tighter loose death dating is possible

### ISSUE THREAD

**Issue by brianfreud** [ShammyLevva/FTAnalyzer#234]
Currently, loose deaths doesn't seem to take birth range dates into consideration.  For example, one person I had with a birth date of "BET 1634 AND 1728", and no death date set.  Ftanalyzer suggested "BET 26 MAY 1744 AND 13 SEP 2021", based solely on the marriage date of "26 May 1744".

"BET 26 MAY 1744 AND 13 SEP 2021" could be tightened up to take that high end of the birth range 1728+110, giving "BET 26 MAY 1744 AND 1838".  It's still a huge range of time between the earliest birth and the latest death dates in those 2 ranges, but it'd be better than before.

**Comment by ShammyLevva** [comment]
Thanks for that. I could have sworn it already did this but clearly doesn’t. Definitely a bug. Same is possibly true for loose births I’ll need to check.

**Comment by brianfreud** [comment]
I just found another case, this time where the loose birth was based on marriage date, but the loose death wasn't range capped properly.  Marriage date "1688"; gave a suggested loose birth between 1628 and 1672.  Children assigned to her born on 23 Jan 1688 and on 24 Nov 1692.  The suggested death date, however, seems only based on the marriage date and the birth date of her first child, without an upper bound or considering the birth of her second child: "BET 23 JAN 1688 AND 19 SEP 2021".

**Comment by ShammyLevva** [comment]
Yeah I suspect it's the special case where the cap is being set as today, and not also checked for a cap to 110 years.

**Comment by ShammyLevva** [comment]
Just looked at code and it should be capped but clearly isn't. Can you send me a sample file to ftanalyzer@gmail.com with a single individual/family that shows this behaviour please. I can then load it up and step through the code to determine why it's not triggering with your data.

**Comment by ShammyLevva** [comment]
Still awaiting example. Unfortunately I am unable to reproduce the error at present.

**Comment by ShammyLevva** [comment]
I'll need to close this issue if I cannot reproduce it.
